style_transfer/fast_style_transfer_net.js

```diff
@@ -207,6 +207,10 @@ export class FastStyleTransferNet {
     this.compiledModel_ = await this.model_.compile(options);
   }
 
+  dispose() {
+    this.compiledModel_.dispose();
```

Comment:
Because `dispose` is only available for polyfill implementation. Please check whether it is not undefined before call it.

Reply:
Thanks, done.